lib/iota.js

```diff
@@ -3,6 +3,7 @@ var makeRequest = require('./utils/makeRequest');
 var api = require("./api/api");
 var Multisig = require('./multisig/multisig');
 
+'use strict';
```

Comment:
Not sure if we want every file to have this, but I saw it at the top of `api.js`, so I assumed that this is the standard.